src/resources/basis.js

```diff
@@ -494,4 +517,11 @@ function basisTranscode(url, data, callback, options) {
     }
 }
 
-export { basisDownload, basisDownloadFromConfig, basisInitialize, basisTargetFormat, basisTranscode };
+export {
+    basisDownload,
+    basisSetDownloadConfig,
```

Comment:
I made this list alphabetical. And you BROKE it! 😢

Reply:
Lol! Without thinking, I just assumed the order here would match the order they appear in the file. I'll fix it.